src/baseline_bots/bots/smart_order_accepter_bot.py

```diff
@@ -278,12 +357,23 @@ def generate_support_proposals(self, comms_obj: MessagesData) -> Dict[str, str]:
                 final_messages[self.game._unit_owner(selected_order[0]).name].append(
                     selected_order[2]
                 )
-            
+
             for recipient in final_messages:
                 # Construct message for each support proposal
-                if not(final_messages[recipient]):
+                if not (final_messages[recipient]):
```

Comment:
remove (parens)

Reply:
I didn't write this but it was caused by the formatter I believe, is it worth it to turn off formatting with comments just to fix that?